el/rtpengine.spec

```diff
@@ -236,15 +249,25 @@ true
 %attr(0750,%{name},%{name}) %dir %{_var}/spool/%{binname}
 # Documentation
 %doc LICENSE README.md debian/changelog debian/copyright
+%{_mandir}/man8/%{binname}.8*
 
 %files kernel
-/%{_lib}/xtables/libxt_RTPENGINE.so
+## /usr/lib64/xtables/libxt_RTPENGINE.so
```

Comment:
Hm, what's the idea behind this change?

Reply:
That artifact  was removed in eea3824725

Reply:
Right, but shouldn't we _drop_ it then here? :)

Reply:
Done and pushed